src/test/scala/chiselTests/BundleLiterals.scala

```diff
@@ -0,0 +1,66 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import chisel3._
+import org.scalatest._
+
+class BundleWithLit(lit: Option[Int]=None) extends Bundle {
+  val internal = lit.map(_.U).getOrElse(UInt(4.W))
+  override def cloneType = new BundleWithLit(lit).asInstanceOf[this.type]
```

Comment:
Is this a mistake? I just realized that cloneType of a lit should not be a lit. This should probably be `new BundleWithLit().asInstanceOf[this.type]`

Reply:
I think that's right, that cloneType should not return a Lit.